guides/07_FAQs.md

```diff
@@ -620,4 +625,40 @@ Meeting join comprises several steps, one of which is establishing a signaling c
 ### How do I receive Amazon Chime SDK channel messages without using the Amazon Chime SDK for JavaScript?
 
 Follow the instructions in the ["Using websockets to receive messages" developer guide](https://docs.aws.amazon.com/chime-sdk/latest/dg/websockets.html#connect-api).
-To sign the URL in Python, use the example code in the [GitHub issue #1241](https://github.com/aws/amazon-chime-sdk-js/issues/1241#issuecomment-830705541).
\ No newline at end of file
+To sign the URL in Python, use the example code in the [GitHub issue #1241](https://github.com/aws/amazon-chime-sdk-js/issues/1241#issuecomment-830705541).
+
+## Mobile SDK
+
+### What does each MeetingSessionStatusCode mean?
+
+The definition of each MeetingSessionStatusCode can be found [here](https://aws.github.io/amazon-chime-sdk-ios/Enums/MeetingSessionStatusCode.html) for iOS and [here](https://aws.github.io/amazon-chime-sdk-android/amazon-chime-sdk/com.amazonaws.services.chime.sdk.meetings.session/-meeting-session-status-code/index.html) for Android.
+
+### How should I handle each MeetingSessionStatusCode in my application?
+MeetingSessionStatusCode provides additional details for the MeetingSessionStatus received for a session. While some codes are used internally for MeetingSessionStatus, their primary use is in callbacks of AudioVideoObserver. Specifically, they are utilized in `AudioVideoObserver.audioSessionDidStopWithStatus(MeetingSessionStatus)`/`AudioVideoObserver.videoSessionDidStartWithStatus(MeetingSessionStatus)`/`AudioVideoObserver.videoSessionDidStopWithStatus(MeetingSessionStatus)` for iOS, and their equivlants for Android. There are also usages in ReplicatedMeetings, they have been documented separately([iOS](https://github.com/aws/amazon-chime-sdk-ios/blob/master/guides/replicated_meetings.md)/[Android](https://github.com/aws/amazon-chime-sdk-android/blob/master/guides/replicated_meetings.md)).
+
+These status codes can be used for logging, debugging, and notification to end users, but in most cases should not be used for any retry behavior, as Chime SDK will already be retrying non-terminal errors. The table below illustrates how each code should be handled.
+
+| MeetingSessionStatusCode | Callbacks | How to handle |
+| -------- | ------- | ------- |
+| OK(iOS/Android) | `audioSessionDidStopWithStatus(MeetingSessionStatus)`  `videoSessionDidStartWithStatus(MeetingSessionStatus)`  `videoSessionDidStopWithStatus(MeetingSessionStatus)` | This code indicates the audio/video sessions on the device have been started/stopped without any errors. When received from AudioVideoObserver callbacks, your application may want to notify your user about the success of session start/stop.
```

Comment:
- We can probably call out why specific codes are present in Android vs iOS SDK for example the `Left` one for builder clarification.
- Probably you may have done so, but, suggest checking the reasoning once with the JS SDK statuses as well and if we need any changes for consistency.

Reply:
I don't have enough info to tell why there are discrepancies between iOS and Android, we just want to document what we have at this moment. There is a backlog to investigate and improve those codes.